### DISCUSSION TITLE
I'm trying to deploy opencti but it seems to have a problem :(

### DISCUSSION THREAD

**Discussion by ElodiaNadine** [OpenCTI-Platform/opencti#3030] [Q&A]
I'm using a VMWare workstation on Kali Linux 2023-1.

I want to deploy opencti using docker, I have tried using portainer and docker-compose, but it seems nothing is working for me. :(

1. I was following the YouTube tutorial for deploying opencti using a portainer; I had already completed the step and had successfully accessed the portainer, but when I attempted to deploy the opencti stack, I received an error message stating that the stack could not be deployed for an unknown reason.

According to the tutorial, they simply copy the raw code of the docker-compose.yml, so I did the same. and I have already changed the.env files (using UUID V4 generator)

```
OPENCTI_ADMIN_EMAIL=admin@opencti.io
OPENCTI_ADMIN_PASSWORD=test123
OPENCTI_ADMIN_TOKEN=3bc7ed48-28e6-45d1-b4d3-14af796b6b66
OPENCTI_BASE_URL=http://localhost:8080
MINIO_ROOT_USER=admin
MINIO_ROOT_PASSWORD=admin
RABBITMQ_DEFAULT_USER=admin
RABBITMQ_DEFAULT_PASS=admin
CONNECTOR_EXPORT_FILE_STIX_ID=2308f441-2ffb-4a57-ab6e-b03dcf2aad80
CONNECTOR_EXPORT_FILE_CSV_ID=ad6ba8ad-d88f-4068-92bf-9c2f58a1e2b8
CONNECTOR_EXPORT_FILE_TXT_ID=9decb192-d671-4861-887b-3ff7e6696f34
CONNECTOR_IMPORT_FILE_STIX_ID=a90de46b-3460-440c-8a1b-69b0e6c844fd
CONNECTOR_IMPORT_DOCUMENT_ID=9750db8f-885e-4f15-b2d3-4be9ded0367e
SMTP_HOSTNAME=localhost
ELASTIC_MEMORY_SIZE=4G    
```

2. I also have followed the documentation tutorial using docker only. but i still cannot access the localhost, and when i check the log it says like this,

{"category":"APP","error":{"context":{"category":"technical","error":"getaddrinfo ENOTFOUND elasticsearch","http_status":500,"reason":"[SEARCH] Search engine seems down"},"message":"A configuration error has occurred","name":"ConfigurationError","stack":"ConfigurationError: A configuration error has occurred\n    at error (/opt/opencti/build/src/config/errors.js:8:10)\n    at ConfigurationError (/opt/opencti/build/src/config/errors.js:61:53)\n    at /opt/opencti/build/src/database/engine.js:175:15\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at searchEngineInit (/opt/opencti/build/src/database/engine.js:165:3)\n    at checkSystemDependencies (/opt/opencti/build/src/initialization.js:130:3)\n    at platformStart (/opt/opencti/build/src/boot.js:164:5)"},"level":"error","message":"[OPENCTI] Platform start fail","timestamp":"2023-03-20T13:34:13.369Z","version":"5.6.2"}
{"category":"APP","error":{"context":{"category":"technical","error":"connect ECONNREFUSED 172.20.0.11:9200","http_status":500,"reason":"[SEARCH] Search engine seems down"},"message":"A configuration error has occurred","name":"ConfigurationError","stack":"ConfigurationError: A configuration error has occurred\n    at error (/opt/opencti/build/src/config/errors.js:8:10)\n    at ConfigurationError (/opt/opencti/build/src/config/errors.js:61:53)\n    at /opt/opencti/build/src/database/engine.js:175:15\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at searchEngineInit (/opt/opencti/build/src/database/engine.js:165:3)\n    at checkSystemDependencies (/opt/opencti/build/src/initialization.js:130:3)\n    at platformStart (/opt/opencti/build/src/boot.js:164:5)"},"level":"error","message":"[OPENCTI] Platform start fail","timestamp":"2023-03-20T13:35:03.993Z","version":"5.6.2"}

before that, I also check the docker ps, and it shows like this

![image](https://user-images.githubusercontent.com/116774812/226369437-9dc91f29-2def-4a8b-83c8-9e72e5328c7e.png)

Can anyone please help me :( i have been stuck like this. I'm new and I wasn't exactly sure how this worked. please enlighten me. Thankyou.

**Comment by LiamWBA** [comment]
Your elastic search memory seems too low. My guess would be that is blocking your sign ins because the elastic search ui can't handle anymore workload

**Reply by cyberintelligenceworld** [reply]
I am having the same issues - I have increased the memory and disk sized too
